src/components/ui/Pool/ZapOutSection.tsx

```
@@ -0,0 +1,123 @@
+            </Button>
+            <Button className="bg_turq" onClick={() => setSliderValue(100)}>
+              <Typography secondary size="xs" fw={700} tt="uppercase">
+                MAX
+              </Typography>
+            </Button>
+          </div>
+          <Button
+            onClick={
+              isConnected
+                ? () => onZapOut(zapAmount, zapTokenAddress as Hex)
+                : () => openConnectModal && openConnectModal()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)